### DISCUSSION TITLE
Collapsible Entry Section Headings

### DISCUSSION THREAD

**Discussion by elfacht** [craftcms/cms#14621] [Ideas] [answered]
With the Entrification plan we're already converting categories to structures etc., but we experience a cluttered list which will get longer and longer. My feature suggestion would be collapsible section headers with a cookie/local storage remembering the staes.

**Before**

![Screen shot 2024-03-19 um 07 25 10](https://github.com/craftcms/cms/assets/529443/973f942e-b6f0-4bdf-a538-f70c1cca8866)

**After**

![Screen shot 2024-03-19 um 07 25 40](https://github.com/craftcms/cms/assets/529443/86bc9950-7c11-4fe3-93da-1bfe7ec7878b)

**Comment by brandonkelly** [comment, marked as answer]
Thanks for posting! This was requested previously via #11340. Follow along there for updates.
